nixos/modules/services/hardware/undervolt.nix

```
@@ -8,12 +8,22 @@ let
     if (limit == null && window == null) then null
     else assert asserts.assertMsg (limit != null && window != null) "Both power limit and window must be set";
       "${toString limit} ${toString window}";
+  hasCliArgs = (
+    # One of the following must be set to enable the service
+    (cfg.coreOffset != null) ||
+    (cfg.gpuOffset != null) ||
+    (cfg.uncoreOffset != null) ||
+    (cfg.analogioOffset != null) ||
+    (cfg.temp != null) ||
+    (cfg.tempBat != null) ||
+    (cfg.turbo != null) ||
+    (cfg.p1.limit != null) ||
```

in light of the assertion below:
```suggestion
    (cfg.p1.limit != null) &&
```